examples/robot_bodybrain_ea/main.py

```diff
@@ -187,7 +186,6 @@ def main() -> None:
         best_robot = find_best_robot(best_robot, population)
 
         logging.info(f"Best robot until now: {best_robot.fitness}")
-        logging.info(f"Genotype pickle: {pickle.dumps(best_robot)!r}")
```

Comment:
Ok, but it's annoying to run this example. Hard to see the best robot performance between generations. Can we stop it from printing?

Reply:
you could simply let it print the best robot in the end
also make sure you are on the newest version on dev branch because i see you stil have the old example structure